test/buy-tshirt.spec.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable protractor/no-describe-selectors */
```

Comment:
I don't understand the link you put, it's saying to set a variable `this.parent` not to disable